app/install/microsoft-azure.md

```diff
@@ -0,0 +1,99 @@
+---
+id: page-install-method
+title: Install - Microsoft Azure
+header_title: Microsoft Azure
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+---
+
+## Table of Contents
+
+<!-- FIXME the list below should be an unordered list, but currently those do not render correctly in this section of the Docs site - depends on https://github.com/Kong/docs.konghq.com/issues/917 -->
+1. [Running Kong on Azure Container Instances](#running-kong-on-azure-container-instances)
+1. [Running PostgreSQL on Azure with Azure Database for PostgreSQL](#running-postgresql-on-azure-with-azure-database-for-postgresql)
+1. [Running Cassandra on Azure with Azure Cosmos Db](#running-cassandra-on-azure-with-azure-cosmos-db)
+
+# Running Kong on Azure Container Instances
+<a href="https://docs.microsoft.com/en-us/azure/container-instances/container-instances-overview" target="blank">Azure Container Instances</a> is a great way to run lightweight containers in a serverless fashion.
+
+Running Kong on Azure Container Instances is super easy:
+
+1. **Provision a data store**
+
+    Provision the data store that you want to use:
+    1. [Running Cassandra on Azure with Azure Cosmos Db](#running-cassandra-on-azure-with-azure-cosmos-db)
+    1. [Running PostgreSQL on Azure with Azure Database for PostgreSQL](#running-postgresql-on-azure-with-azure-database-for-postgresql)
+
+1. **Open the Cloud Shell or Azure CLI**
+
+1. **Run the migrations**
+
+    ```bash
```

Comment:
@darrenjennings I was thinking I'd see the awesomesauce from https://github.com/Kong/docs.konghq.com/pull/969 here, but when I run the docs site locally, and visit this page, and triple-click on the code sample, the leading $ prompt gets selected - any ideas?

![select-prompt](https://user-images.githubusercontent.com/94904/47884673-ea708e00-ddee-11e8-971e-400db2161f63.gif)